tests/integration_tests/dashboards/superset_factory_util.py

```diff
@@ -145,11 +160,14 @@ def create_datasource_table_to_db(
 def create_datasource_table(
     name: Optional[str] = None,
     db_id: Optional[int] = None,
+    database: Optional[Database] = None,
     owners: Optional[List[User]] = None,
 ) -> SqlaTable:
-    name = name or random_str()
-    owners = owners or []
-    db_id = db_id or create_database_to_db(name=name + "_db").id
+    name = name if name is not None else random_str()
+    owners = owners if owners is not None else []
```

Comment:
I see a lot of these changes from an 'or 'statement to explicitly checking for None. It doesn't look like there are other falsy values that you would have, I'm curious what the advantage of this change is?